src/fast++-read_input.cpp

```diff
@@ -1335,9 +1340,11 @@ bool read_spectra(const options_t& opts, input_state_t& state) {
     state.flux = replicate(0.0f, ngal, nplam+nslam);
     state.eflux = replicate(finf, ngal, nplam+nslam);
 
-    for (uint_t i : range(ngal)) {
-        state.flux(i,_-(nplam-1)) = pflx(i,_);
-        state.eflux(i,_-(nplam-1)) = perr(i,_);
+    if (nplam > 0){
```

Comment:
```suggestion
    if (nplam != 0) {
```

Reply:
Again, your code is fine. It's just clearer to explicitly state we don't want ```nplam``` to be zero rather than ask it be strictly positive.

Reply:
Hi @cschreib. Thanks for reviewing the code. I've accepted all the suggestions above. I think that also applies all the formatting requests you made as well. Let me know if not.

Reply:
Great thanks! It did work. However there are still a few styling modifications to make that I could not do easily with the github interface (it won't let me suggest changes to more than one line at a time, which is cumbersome when I need to correct the indentation). Can you please check and fix the following issues:

* Indent with 4 spaces.
* Leave a blank space after the parenthesis of an ```if``` clause, for example replace ```if (a){``` by ```if (a) {```.
* Put the ```else``` keyword on the same line as the closing ```}``` of the ```if```, and leave a blank space after ```else```.

If you make these changes in your local repository and push them to the master branch of your fork, the changes will automatically be carried over to this pull request.